modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -1209,6 +1209,12 @@ class NDB_Form_User_Accounts extends NDB_Form
                 $errors['examiner_group'] = "Please specify if examiner is a ".
                     "radiologist";
             }
+            if (preg_match("/^ex_[1-9]+$/", $k)
+                && $values['examiner_radiologist'] !== ''
+                && $values['examiner_pending'] == ''
```

Comment:
basically this if and the one before check is any of the examiner checkboxes are checked and throw an error if they are but the pending approval is still set to null

The radiologist and pending_approval should only be required if the examiner checkboxes are used

due to the way checkboxes are submitted or not submitted this was the clearest way I can think of to do this check